### PR TITLE
Fix token smushing performance issue

### PR DESCRIPTION
**Pull request by kag0**
I have no comments, only shame

### CODE REVIEW

**Comment by tminglei** [comment]
Merged. Thanks!

**Comment by tminglei** [comment]
Released in slick-pg `v0.20.2`.

File: core/src/main/scala/com/github/tminglei/slickpg/utils/PgTokenHelper.scala [hunk 1]
```diff
@@ -79,15 +79,24 @@ object PgTokenHelper {
...
+    case GroupToken(mList) => 
+      smush(Vector.empty, mList)
+        // I don't know why, but CompositeConverter.fromToken is going to 
+        // throw an IllegalArgumentException: argument type mismatch 
+        // if you don't make this a list right here and now
+        // no, getChildren(tokens).toList will not work
+        .toList 
+        .filterNot(_ == null)
+        .filterNot(_.isInstanceOf[Border])
+        .filterNot(_ == Comma)
```
Comment (longshorej) [inline core/src/main/scala/com/github/tminglei/slickpg/utils/PgTokenHelper.scala:99]: Each of these `filterNot`s allocates a new collection - would it be worth combining them into one invocation in a future PR?

Comment (tminglei) [reply core/src/main/scala/com/github/tminglei/slickpg/utils/PgTokenHelper.scala:99]: Yes, combining will be better. I'll do it.

**Pull request merged**